src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -215,6 +216,136 @@ def add_imaging_plane(nwbfile: NWBFile, metadata: dict, imaging_plane_index: int
     return nwbfile
 
 
+def add_plane_segmentation(
+    segmentation_extractor: SegmentationExtractor,
+    nwbfile: NWBFile,
+    metadata: Optional[dict],
+    plane_segmentation_index: int = 0,
+    image_segmentation_name: str = None,
+    iterator_options: Optional[dict] = None,
+    compression_options: Optional[dict] = None,
+) -> NWBFile:
```

Comment:
Thank you @h-mayorquin, I followed your comments and changed the `add_plane_segmentation` method to be more similar to the `add_electrodes`. However, there are some arguments that I cannot lose at the moment (which makes it different from `add_electrodes`):
- `plane_segmentation_index`: similar to `imaging_plane_index` in `add_imaging_plane` method
- `image_segmentation_name`: plane segmentation is added to this image_segmentation (if this is also created within this function then this is not needed, but I'm not sure)
- `iterator_options`, `compression_options`: these are configurations for H5DataIO)

Reply:
Yes, that makes sense that they are different in this way. I wonder about `image_segmentation_name`. I think this should be in the default metadata:
https://github.com/catalystneuro/neuroconv/blob/c9170f6ef33719d625d7281cbbc58cbb4e0f6b6a/src/neuroconv/tools/roiextractors/roiextractors.py#L77

What do you think?

Reply:
Yes, this makes sense. I changed in `get_default_ophys_metadata()` to have a default name that can be fetched.

Reply:
Great!